core/src/main/java/org/zenoss/zep/zing/ZingEvent.java

```diff
@@ -182,13 +189,56 @@ public Event toZingEventProto() {
             b.putDimensions(ZingConstants.FINGERPRINT_KEY, ZingUtils.getScalarValueFromObject(this.fingerprint));
             b.putDimensions(ZingConstants.UUID_KEY, ZingUtils.getScalarValueFromObject(this.uuid));
 
-            //-----------
-            //  Metadata
-            //-----------
-            if (!ZingUtils.isNullOrEmpty(this.severity))
-                b.putMetadata( ZingConstants.SEVERITY_KEY, ZingUtils.getScalarArray(this.severity));
-            if (!ZingUtils.isNullOrEmpty(this.status))
+            //--------------------------
+            //  Status Map
+            //--------------------------
+            Map<String, Status> statusMap = new HashMap<String, Status>();
```

Comment:
make statusMap and severityMap into static members of ZingEvent and populate them with a static initializer.